tools.templates/org.eclipse.tools.templates.core/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Automatic-Module-Name: org.eclipse.tools.templates.core
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.tools.templates.core
-Bundle-Version: 1.3.0.qualifier
+Bundle-Version: 2.0.0.qualifier
```

Comment:
I didn't bump this as there is no API change on this plugin. But I can see the argument for keeping it consistent with the others

Reply:
Thanks @Kummallinen